easycord/_bot_guild.py

```
@@ -0,0 +1,175 @@
+        caches it. Subsequent calls reuse the cached webhook.
+
+        Example::
+
+            await bot.send_webhook(CHANNEL_ID, "Hello from a webhook!")
+        """
+        if channel_id not in self._webhooks:  # type: ignore[attr-defined]
+            channel = self.get_channel(channel_id)
+            if not isinstance(channel, discord.TextChannel):
+                raise RuntimeError(
+                    f"Channel {channel_id} is not a text channel or was not found in cache"
+                )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)